src/main/java/seedu/address/model/issue/Issue.java

```diff
@@ -131,7 +131,11 @@ public String toString() {
 
     @Override
     public int compareTo(Issue other) {
-        return timestamp.compareTo(other.timestamp);
```

Comment:
I think it would be better to change this form to use the standard if-else syntax. There are 3 reasons for suggesting this:

1. It improves readability of what the function is doing.
2. The code-base throughout does not use ternary operators in the `compareTo` or `equals` function. They typically write it out in full form.
3. The module's coding standards require if-else type statements to take a certain form: https://se-education.org/guides/conventions/java/intermediate.html#layout

Thus, I would recommend changing it.